tests/test_input.py

```diff
@@ -5,6 +5,13 @@
 
 
 class TestReader:
+    def test_incorrect_file_name_type(self):
+        with pytest.raises(TypeError):
+            reader = ReadData(None)
+
+        with pytest.raises(TypeError):
+            reader = ReadData(file_name=["datasets/encoding/test.csv"])
+
     def test_file_name(self):
```

Comment:
`test_incorrect_file_type` looks like a better name for this test.

Reply:
Okay, will do the change